zinc/lib/zinc_types/zinc_types_intf.ml

```diff
@@ -163,8 +163,8 @@ module type S = sig
     and adt =
       | MakeRecord of int
       | RecordAccess of int
-      | MakeVariant of variant_label
-      | MatchVariant of (variant_label * t) list
+      | MakeVariant of label
+      | MatchVariant of t LMap.t
```

Comment:
Should we change `MakeRecord` and `RecordAccess` to use `label` instead of `int` in a future PR?

Reply:
changed